brainiak/isfc.py

```diff
@@ -84,7 +84,6 @@ def isc(D, collapse_subj=True, return_p=False, num_perm=1000,
         n_perm = num_perm
     else:
         n_perm = 0
-
     ISC = np.zeros((n_vox, n_subj, n_perm + 1))

```

Comment:
This still has the n_perm parameter in the array. Is this okay?

Reply:
Yes, it is. I want to make sure the modification on ISFC works before also changing ISC. But, yeah, I can also change ISC now